docs/dxp/7.x/en/developing-applications/core-frameworks/configurable-applications/resources/liferay-c6f7.zip/c6f7-web/src/main/resources/META-INF/resources/configuration.jsp

```diff
@@ -0,0 +1,98 @@
+<%@ page import="com.liferay.portal.kernel.util.Constants" %>
+
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+
+<%@ taglib uri="http://java.sun.com/portlet_2_0" prefix="portlet" %>
+
+<%@ taglib uri="http://liferay.com/tld/aui" prefix="aui" %><%@
+taglib uri="http://liferay.com/tld/portlet" prefix="liferay-portlet" %><%@
+taglib uri="http://liferay.com/tld/theme" prefix="liferay-theme" %><%@
+taglib uri="http://liferay.com/tld/ui" prefix="liferay-ui" %>
+
+<%@ page import="com.acme.c6f7.web.internal.portlet.MessageDisplayConfiguration" %><%@
+page import="com.liferay.petra.string.StringPool;" %><%@
+page import="com.liferay.portal.kernel.util.Validator" %>
+
+<liferay-theme:defineObjects />
+
+<portlet:defineObjects />
+
+<%
+MessageDisplayConfiguration messageDisplayConfiguration =
+	(MessageDisplayConfiguration)
+	renderRequest.getAttribute(MessageDisplayConfiguration.class.getName());
+String fontFamily = StringPool.BLANK;
+String fontColor = StringPool.BLANK;
+String fontSize = StringPool.BLANK;
+
+if (Validator.isNotNull(messageDisplayConfiguration)) {
```

Comment:
SF complains about using `Validator.isNotNull` because it is only intended for using with Long, Serializable, and String objects. So you just have to do a regular null check ...

`if (messageDisplayConfiguration != null) {`